src/app.jsx

```diff
@@ -33,16 +33,16 @@ class App extends React.Component {
         console.log('ERROR IN AXIOS GET REQUEST', error);
       });
 
-    axios.get(`http://${this.reviewsip}:3004/averagereview/` + this.state.id)
-      .then((response) =>
-        that.setState({
-          avgReviews: response.data.averageReviews,
-          totalReviews: response.data.totalReviews
-        })
-      )
-      .catch((error) =>
-        console.log('ERROR GETTING REVIEW', error)
-      );
+    // axios.get(`http://${this.reviewsip}:3004/averagereview/` + this.state.id)
```

Comment:
looks good but why is it commented out?

Reply:
This is calling an external service which is no longer running.